### PR TITLE
Use ephemeris to handle shed tools for workflows.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: docs/presentations/2016_workflows.html [hunk 1]
```diff
@@ -14,18 +16,26 @@
...
-# Planemo: A Scientific Workflow SDK
+class: title
+# Planemo
+## ~~A Galaxy Tool SDK~~
+## A Scientific Workflow SDK
 John Chilton, Aysam Guerler, and the Galaxy Team
 
+???
+
+Oh look, I get to follow Björn again. That guy ruins the curve - when
```
Comment (martenson) [inline docs/presentations/2016_workflows.html]: I had to google for a bit and still do not understand the phrase 'ruin the curve' correctly I am afraid. Non-native speakers might have problems catching this.

Comment (nsoranzo) [inline docs/presentations/2016_workflows.html]: This is a reference to https://en.wikipedia.org/wiki/Grading_on_a_curve , so "to ruin the curve" is a sarcastic way of saying that somebody did something that other people will have a hard time competing with.

Comment (jmchilton) [inline docs/presentations/2016_workflows.html]: Okay - I'll try to come up with another way to say that for you freedom haters. Thanks!

Comment (peterjc) [inline docs/presentations/2016_workflows.html]: Now you're using other Americanisms just for fun, aren't you?

Comment (martenson) [inline docs/presentations/2016_workflows.html]: @nsoranzo Americanism isn't an actual thing - let's just call it post-medieval Europeanism.

**Pull request merged**